Audacia.Log.AspNetCore/README.md

```diff
@@ -67,7 +67,55 @@ public IConfiguration Configuration { get; set; }
 public void ConfigureServices(IServiceCollection services)
 {
 	services.ConfigureActionContentLogging(Configuration);
-	services.AddControllers(x => x.Filters.Add<LogActionFilterAttribute>());
+    services.AddClaimsTelemetry();
+	services.AddControllers(x => x.Filters.Add<LogClaimsActionFilterAttribute>());
+}
+```
+
+#### AddClaimsTelemetry()
+This extension method is overloaded and if you want to pass in your own implementation of "CustomAdditionalClaimsTelemetryProvider" to select which properties you would like to pass on, do it like below:
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+	services.ConfigureActionContentLogging(Configuration);
+ .AddClaimsTelemetry(new CustomAdditionalClaimsTelemetryProvider((user) => 
+  {
+      return new List<(string Name, string Data)>
+      {
+          ("customproperty", user.Claims.Where(c => c.Type == "customproperty").Single().Value)
+      };
+  }))
+	services.AddControllers(x => x.Filters.Add<LogClaimsActionFilterAttribute>());
+}
+```
+
+#### RequestBodyActionLogFilter
+This filter can be registered to include logs for the beginning and end of each Action Method. This only includes request parameters as well as details of the response such as the type of model returned. Register it like so:
+
+```csharp
+using Audacia.Log.AspNetCore;
+
+public IConfiguration Configuration { get; set; }
+
+public void ConfigureServices(IServiceCollection services)
+{
+	  services.ConfigureActionContentLogging(Configuration);
+      services.AddRequestBodyTelemetry();

```

Comment:
`nit:` Can the indentation be fixed here?

Reply:
Will do.